tests/cli/test_utils.py

```diff
@@ -323,3 +325,22 @@ def test_validate_assistant_id_in_config(config_file: Text) -> None:
 
     # reset input files to original state
     rasa.shared.utils.io.write_yaml(copy_config_data, config_file, True)
+
+
+def test_validate_assistant_id_in_config_preserves_comment() -> None:
+    config_file = "data/test_config/config_no_assistant_id_with_comments.yml"
+    copy_config_data = copy.deepcopy(rasa.shared.utils.io.read_yaml_file(config_file))
+
+    rasa.cli.utils.validate_assistant_id_in_config(config_file)
+    config_data = rasa.shared.utils.io.read_yaml_file(config_file)
+    yaml = YAML()
+    buffer = io.StringIO()
+    yaml.dump(config_data, buffer)
+    config_file_content = buffer.getvalue()
+
+    comment = "# Random comments line {}"
+    for i in range(1, 6):
+        assert comment.format(i) in config_file_content
```

Comment:
Is there a way to check that the comments are kept in the line that's expected, rather than moved around?

Reply:
I'm not sure but we don't even wanna test that cos a yaml file won't stay exactly the same after an update.
e.g
```
x: [
      ["a", "b", "c"], 
      ["d", "e", "f"]
   ]
```

is re-written as 
`x: [["a", "b", "c"], ["d", "e", "f"]]`

Reply:
Ah I see, okay.

Reply:
One thing I forgot to ask for is a changelog entry 😄

Reply:
Yeah I was gonna confirm if I needed one.
Your review speed was faster than lightning 😄 🚀 

I'll add it now 👍🏾